### PR TITLE
Adding support for vb.net (.vbhtml) to CompositePrecompiledMvcEngine

### PR DESCRIPTION
**Pull request by CZEMacLeod**
Fix nuspec against issues #5 and #118
Bump rev to 2.4.9

### CODE REVIEW

File: RazorGenerator.Mvc/PrecompiledMvcEngine.cs [hunk 1]
```diff
@@ -33,53 +33,46 @@ public PrecompiledMvcEngine(Assembly assembly, string baseVirtualPath)
...
+                "~/Areas/{2}/Views/Shared/{0}.vbhtml",
             };
+        }
 
-            base.AreaMasterLocationFormats = new[] {
-                "~/Areas/{2}/Views/{1}/{0}.cshtml", 
-                "~/Areas/{2}/Views/{1}/{0}.vbhtml", 
-                "~/Areas/{2}/Views/Shared/{0}.cshtml", 
-                "~/Areas/{2}/Views/Shared/{0}.vbhtml", 
+        public static string[] DefaultLocationFormats()
```
Comment (pranavkm) [inline RazorGenerator.Mvc/PrecompiledMvcEngine.cs]: Could you make these regular properties or readonly fields? It's a bit unusual to see methods that don't follow the Get \ Set \ Do* format.

Comment (CZEMacLeod) [inline RazorGenerator.Mvc/PrecompiledMvcEngine.cs]: Sorry - I don't use Git very often and don't think that committed the way I wanted it to, but that is the changes as requested to make these readonly fields.

**Comment by CZEMacLeod** [comment]
Changes as requested...

**Comment by reijerh** [comment]
Any chance you'll be able to merge this in the near future? The dependency install issue (https://github.com/RazorGenerator/RazorGenerator/issues/118) breaks the build everytime we install packages :(.

**Comment by CZEMacLeod** [comment]
It's been approved - not sure what the schedule for actually bringing this in and releasing an update is though. I have a private build / package I keep on myget with these changes as I had the same problem with the package dependencies.

**Comment by davidebbo** [comment]
@pranavkm looked like you reviewed this earlier. Any reason not to merge in now?

**Comment by pranavkm** [comment]
Nope, should be good to merge it in. Just dropped off my radar

**Comment by davidebbo** [comment]
Cool. What specific NuGet packages need to be published? The version was already bumped to 2.4.9, so all that's left is to push to nuget.

**Comment by pranavkm** [comment]
Think it's only `RazorGenerator.Mvc`.

**Comment by davidebbo** [comment]
Ok, I pushed it. @reijerh @CZEMacLeod can you verify with 2.4.9 on nuget.org? Thanks!

**Comment by CZEMacLeod** [comment]
Tested it in place of my locally built version/package and it all looks good.

**Pull request merged**